src/common/store.ts

```diff
@@ -2,12 +2,14 @@ import { configureStore } from '@reduxjs/toolkit';
 import { combineReducers } from 'redux';
 
 import checkedRevisions from '../reducers/CheckedRevisions';
+import results from '../reducers/ResultsSlice';
 import search from '../reducers/SearchSlice';
 import selectedRevisions from '../reducers/SelectedRevisions';
 
 const reducer = combineReducers({
-  search,
   checkedRevisions,
+  results,
```

Comment:
[question] would we have other types of results?
maybe this should be renamed to compareResults ? 🤷‍♀️